test/evergreen.yml

```diff
@@ -1027,6 +1027,16 @@ tasks:
       - func: "fetch artifacts"
       - func: "unit test"
 
+  - name: unit-test-long
+    tags: ["unit_test"]
```

Comment:
Be cautious about setting this tag :) We only want this task to be executed under `ubuntu1804` build variant, for now.

Reply:
Ah yes I get your point. Will make the change.